hummingbot/connector/derivative/orderly_perpetual/orderly_perpetual_user_stream_data_source.py

```
@@ -0,0 +1,418 @@
+            # Connect to private WebSocket
+            await ws.connect(
+                ws_url=ws_url,
+                ping_timeout=CONSTANTS.HEARTBEAT_TIME_INTERVAL
+            )
+            
+            self.logger().info(f"[WEBSOCKET] Successfully connected to private WebSocket: {ws_url}")
+
+            # Authenticate the connection
+            await self._authenticate(ws)
+
+            return ws
```

Ruff (0.14.3)

Consider moving this statement to an `else` block

(TRY300)